weather/views.py

```diff
@@ -1,9 +1,18 @@
+from os import getenv
+import requests
+import json
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, JsonResponse
 
 # Create your views here.
 def index(request):
     return render(request, "weather/index.html", locals())
 
 def point_location(request, lat, lon):
+    base_url = getenv("INTEROP_URL")
+    url = f"{base_url}/point/{lat}/{lon}"
+    r = requests.get(url)
+    # TODO: Put some error handling here
```

Comment:
What feels like the blocker for this todo? Is it because we don't have logging and error handling set up in Django yet (need a ticket for that?) or is it because it is not clear what the logic should be if an error occurs?

Reply:
We do not have error handling set up at all yet here, so we do need a ticket for that. Preliminary issue created [here](https://vlab.noaa.gov/gitlab-licensed/NWS/Systems/DIS/Weather.gov-2.0/-/issues/104).